.github/workflows/check_issue_label.yml

```
@@ -0,0 +1,115 @@
+              auth: process.env.PAT_TOKEN
+            });
+            // ======================
+            // SECURE INPUT HANDLING
+            // ======================
+            const sanitizedLabel = (context.payload.label?.name || '').trim();
+            const actorUsername = (context.actor || '').trim().toLowerCase();
+            const issueNumber = Number(context.issue.number) || 0;
+            const { owner, repo } = context.repo;
+            // Validation checks
+            if (!/^[\w\s-:]+$/i.test(sanitizedLabel)) {
+              throw new Error(`Invalid label format: ${sanitizedLabel}`);
+            }
+            if (!/^[a-z\d-]+$/.test(actorUsername)) {
+              throw new Error(`Invalid username: ${actorUsername}`);
+            }
+            if (!Number.isSafeInteger(issueNumber) || issueNumber < 1) {
+              throw new Error(`Invalid issue number: ${issueNumber}`);
+            }
+            // ======================
+            // TEAM MEMBERSHIP CHECK
+            // ======================
+            async function getTeamMembers() {
+              try {
+                const result = await octokit.graphql({
+                  query: `query ($org: String!, $teamSlug: String!) {
+                    organization(login: $org) {
+                      team(slug: $teamSlug) {
+                        members(first: 100) {
+                          nodes { login }
+                        }
+                      }
+                    }
+                  }`,
+                  org: 'oppia',
+                  teamSlug: 'oppia-good-first-issue-labelers'
+                });
+                return new Set(
+                  result.organization.team.members.nodes
+                    .map(node => node.login.toLowerCase())
+                );
+              } catch (error) {
+                console.error('Team membership check failed:', error);
+                return new Set(); // Fail closed
```

This currently seems to return an empty set in case of an error, but would it be better to throw an error to stop execution rather than failing silently?